extract/test/test_bbox.py

```diff
@@ -1,16 +1,16 @@
-from extract.bbox import BBoxFactoryMixin
+from extract.bbox import BBoxMixin
 
 
-class TestBboxClass(BBoxFactoryMixin):
+class TestBboxClass(BBoxMixin):
```

Comment:
In this case, `TestBboxClass` only inherits from BBoxMixin.

Normally you'd use a Mixin where some other class is providing the bulk of the functionality, and the Mixin just adds on an extra little bit.

````python
class TextOtherClassWithBbox(OtherClass, BBoxMixin):
````

Reply:
In this case I was mostly writing a test to make sure the inheritance model I was using actually worked haha